index.js

```diff
@@ -2,13 +2,18 @@ var gulp = require('gulp');
 var elixir = require('laravel-elixir');
 var karma = require('karma').server;
 var _ = require('underscore');
+var gutils = require('gulp-util');
 
 var task = elixir.Task;
 
 elixir.extend('karma', function(args) {
+    if (! _.contains(['watch', 'tdd'], gutils.env._)) return;
```

Comment:
This seems to always be true for me? `gutils.env._` returns an array. Not sure if contains can handle that?
`_.contains(['watch', 'tdd'], ['tdd'])` returns false.
`gutils.env._[0]` and it works. Not a good solution though.